README.md

```diff
@@ -138,6 +138,35 @@ And in [vendor.ts](https://github.com/preboot/angular2-webpack/blob/master/src/v
 import 'bootstrap/dist/css/bootstrap.css';
 ```
 
+#### How to include external js library such as jQuery ?
+
+Just install the lib and import the js file in [vendor.ts](https://github.com/preboot/angular2-webpack/blob/master/src/vendor.ts). For example this is how to do it with bootstrap:
+
+```sh
+npm install jquery --save
+```
+
+Note: Skip the npm install step right above, if you aleady installed bootstrap@next as it installs jQuery already.
```

Comment:
I think it is better to still installing it like that so you get one "fixed" version in the root of node_modules instead of depending on one that comes from another library.